standalone/src/init.ts

```diff
@@ -134,7 +134,7 @@ function connectCall(
 }
 
 async function fetchChannelData(store: Store, channelID: string) {
-    const resp = await Client4.doFetch<CallChannelState>(
+    const resp = await RestClient.fetch<CallChannelState>(
```

Comment:
This, and all the Client4 -> RestClient is because now `Client4.doFetch` is protected and `Client4.doFetchWithResponse` is private. So subclassed them to at least let us use doFetch.